packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -1863,6 +1863,34 @@ void main() {
     expect(tester.getTopRight(find.byKey(pKey)).dx, tester.getTopLeft(find.text('text')).dx);
   });
 
+  testWidgets('InputDecorator error', (WidgetTester tester) async {
+    // when error is defined
+    await tester.pumpWidget(
+      buildInputDecorator(
+        decoration: const InputDecoration(
+          error: Text(
+            'error',
+            style: TextStyle(fontSize: 22.0),
+          ),
+        ),
+      ),
+    );
+    //When errorText and error is defined then error will be shown
```

Comment:
I think after this first `buildInputDecorator` you should also verify some value.